src/swap/saga.ts

```diff
@@ -124,30 +140,47 @@ export function* swapSubmitSaga(action: PayloadAction<SwapInfo>) {
       TAG,
       `Approving ${amountToApprove} of ${sellTokenAddress} for address: ${allowanceTarget}`
     )
+    const beforeApproveTimestamp = Date.now()
     yield call(
       sendApproveTx,
       sellTokenAddress,
       amountToApprove,
       allowanceTarget,
       swapApproveContext
     )
+    const afterApproveTimestamp = Date.now()
+    sendApprovalElapsedTimeInMs = afterApproveTimestamp - beforeApproveTimestamp
 
     // Execute transaction
     yield put(swapExecute())
     Logger.debug(TAG, `Starting to swap execute for address: ${walletAddress}`)
 
+    const beforeSwapExecutionTimestamp = Date.now()
     yield call(
       handleSendSwapTransaction,
       { ...action.payload.unvalidatedSwapTransaction },
       swapExecuteContext
     )
+    const afterSwapExecutionTimestamp = Date.now()
+    sendSwapElapsedTimeInMs = afterSwapExecutionTimestamp - beforeSwapExecutionTimestamp
```

Comment:
also not sure why we want this 😅 i thought only quoteToTransactionElapsedTimeInMs would be interesting...